101-documentdb-account-create-multi-region-account/README.md

```diff
@@ -7,9 +7,9 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-This template will create a DocumentDB Database Account with the two specifed regions, the provided name, and the Offer Type set to ***Standard***.
+This template will create a CosmosDB Database Account with the two specifed regions, the provided name, and the Offer Type set to ***Standard***.
 
 By not setting the optional Default Consistency Level parameter, the account will be created with the default consistency level of ***Session***.
```

Comment:
This additional content from the original readme should be included in all of the other readmes as well (as much as is applicable).